androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -83,6 +89,13 @@ fun EnhancedVideoPlayer(
 
     val isFullScreen = orientation == Configuration.ORIENTATION_LANDSCAPE
     var isSettingsOpen by rememberSaveable { mutableStateOf(false) }
+    val autoQualityTrack by remember {
```

Comment:
Why not `rememberSaveable` for all of them?

Reply:
Use it at least for `trackQualityOptions`. The rest, leave as it is and we implement a saver function later on if necessary

Reply:
Done!